ckanext/ckanext-apicatalog_scheming/ckanext/apicatalog_scheming/schemas/organization.json

```diff
@@ -5,27 +5,27 @@
   "fields": [
     {
       "field_name": "title_translated",
-      "label": "Title",
+      "label": "Organization name",
+      "extra_label": "Organization name",
+      "input_title": ["Name in Finnish", "Name in Swedish", "Name in English"],
       "preset": "fluent_core_title_translated",
       "only_default_lang_required": true,
-      "form_placeholder": "My Organization",
+      "form_placeholder": "Organization name",
       "form_languages": ["fi", "sv", "en"],
+      "show_url": true,
       "display_snippet": null
     },
-    {
-      "field_name": "name",
-      "label": "URL",
-      "validators": "not_empty unicode name_validator group_name_validator",
-      "form_snippet": "slug.html",
-      "form_placeholder": "my-theme"
-    },
```

Comment:
Name is mandatory field in CKAN, it cannot be removed, otherwise you can't create new organizations.

Reply:
Added that field back to the schema and fixed the designs of new and edit forms to match